modules/database/src/std/filters/dbnd.c

```diff
@@ -81,8 +81,8 @@ static db_field_log* filter(void* pvt, dbChannel *chan, db_field_log *pfl) {
         status = dbFastGetConvertRoutine[pfl->field_type][DBR_DOUBLE]
                  (localAddr.pfield, (void*) &val, &localAddr);
         if (!status) {
-            send = 0;
-            recGblCheckDeadband(&my->last, val, my->hyst, &send, 1);
+            send = pfl->mask & ~(DBE_VALUE|DBE_LOG);
+            recGblCheckDeadband(&my->last, val, my->hyst, &send, pfl->mask & (DBE_VALUE|DBE_LOG));
             if (send && my->mode == 1) {
```

Comment:
@ralphlange Should this conditional change as well?  eg. into
```c
if ((send & (DBE_VALUE|DBE_LOG)) && my->mode == 1) {
```

Reply:
Hi,

Could you test once more, because I thought it worked fine, but it seems that the dbnd does not work any more. I get now all changes, when I replace dbnd.c with this patch